tool/internal_pkg/generator/generator.go

```diff
@@ -34,16 +34,18 @@ const (
 	KitexGenPath = "kitex_gen"
 	DefaultCodec = "thrift"
 
-	BuildFileName       = "build.sh"
-	BootstrapFileName   = "bootstrap.sh"
-	ToolVersionFileName = "kitex_info.yaml"
-	HandlerFileName     = "handler.go"
-	MainFileName        = "main.go"
-	ClientFileName      = "client.go"
-	ServerFileName      = "server.go"
-	InvokerFileName     = "invoker.go"
-	ServiceFileName     = "*service.go"
-	ExtensionFilename   = "extensions.yaml"
+	BuildFileName                   = "build.sh"
+	BootstrapFileName               = "bootstrap.sh"
+	ToolVersionFileName             = "kitex_info.yaml"
+	HandlerFileName                 = "handler.go"
+	MainFileName                    = "main.go"
+	ClientFileName                  = "client.go"
+	ServerFileName                  = "server.go"
+	InvokerFileName                 = "invoker.go"
+	ServiceFileName                 = "*service.go"
+	ExtensionFilename               = "extensions.yaml"
+	ServerWithMultiServiceFileName  = "server_with_multi_service.go"
+	InvokerWithMultiServiceFileName = "invoker_with_multi_service.go"
```

Comment:
It seems that the codegen module has changed a lot. I need to double-check it.

Reply:
@HeyJavaBean You may see my test prs: [[kitex-tests](https://github.com/cloudwego/kitex-tests/pull/34)], [[kitex-examples](https://github.com/cloudwego/kitex-examples/pull/76)]